web/gui/dashboard_info.js

```diff
@@ -267,6 +267,12 @@ netdataDashboard.menu = {
         info: 'Performance metrics for <b>RetroShare</b>. RetroShare is open source software for encrypted filesharing, serverless email, instant messaging, online chat, and BBS, based on a friend-to-friend network built on GNU Privacy Guard (GPG).'
     },
 
+    'riak': {
```

Comment:
`riak_kv`

@vlvkobal 	please remind me  - do we have any problems with `_` underscore in names ? i recall you faced some issues

Reply:
@ilyam8 Doesn't this apply to the metrics though? The metrics are namespaced under `riak.` currently (e.g. `riak.kv`, `riak.dt` and so on). I tested this with `riak_kv` earlier but I recall the information here wasn't applied anymore

Reply:
> I tested this with riak_kv earlier but I recall the information here wasn't applied anymore

That is why i summon @vlvkobal 	:smiling_imp:

Reply:
Thanks. So we need to remove `_` from the name. How about to name it simply `riakkv` @jchristgit?

Reply:
Yup, renamed in bfcaf640dbfd7ac7afba8bcf98f26bed110803b5.